static/src/js/actions/errors.js

```diff
@@ -59,12 +61,37 @@ export const handleError = ({
   // defaulting to the `message` argument provided to this action
   const [defaultErrorMessage = message] = parsedError
 
-  dispatch(addError({
-    id: requestId,
-    title: `Error ${verb} ${resource}`,
-    message: defaultErrorMessage,
-    notificationType
-  }))
+  // Use a custom error banner when the fetchSubscriptions action throws the error
+  if (action === 'fetchSubscriptions') {
```

Comment:
Can you check this error matches when logged out. When user is logged out CMR will fail but, I don't think we make a subscriptions call at all

Reply:
Updated it to also catch the `fetchCollections` error action as well when logged out now